lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
It's nested in `(function() { /* code */ })()`

https://github.com/avdg/UglifyJS2/blob/00b760363e529284e3077d924079ebaa915de7dd/lib/output.js#L394